<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="SIPl-DT-PS" failures="1" tests="2" name="Test" time="35.648" errors="0" timestamp="2023-11-23T16:36:19 IST">
  <testcase classname="Softude.Hyperlocology.Tests.ParentCompaniesTest" name="addParentCompany" time="12.106"/>
  <testcase classname="Softude.Hyperlocology.Tests.ParentCompaniesTest" name="wrongEmailErrorValidation" time="12.159">
    <failure type="java.lang.AssertionError" message="expected [This value seems to be invalid.] but found [This value should be a valid email.]">
      <![CDATA[java.lang.AssertionError: expected [This value seems to be invalid.] but found [This value should be a valid email.]
at Softude.Hyperlocology.Tests.ParentCompaniesTest.wrongEmailErrorValidation(ParentCompaniesTest.java:25)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 35 stack frames]]>
    </failure>
  </testcase> <!-- wrongEmailErrorValidation -->
</testsuite> <!-- Test -->
